internal/compiler/generator/rust.rs

```diff
@@ -302,6 +302,8 @@ pub fn generate(doc: &Document) -> TokenStream {
         #[allow(clippy::erasing_op)]
         #[allow(clippy::approx_constant)] // We may get those from .slint inputs!
         #[allow(clippy::eq_op)] // The generated code will compare/subtract/etc. equal values
+        #[allow(clippy::redundant_clone)] // The generated code will err on the side of cloning too
+                                          // much
```

Comment:
Do you know what do we clone?  
Long term we should try to avoid it,  but it's ok to have it for now.